pillar/docs.sls

```diff
@@ -1,3 +1,11 @@
+network:
+  host_id: ocp19
+  ipv4: 178.79.135.174
+  #ipv6: 2001:db8::19
```

Comment:
This is commented out until we sort out Linode's new policies?

Reply:
Yes indeed. We will provision IPv6 addresses after addressing the Linode IPv6 issues